build/Dependencies.props

```diff
@@ -13,10 +13,10 @@
 
   <!-- Other/Non-Core Product Dependencies -->
   <PropertyGroup>
-    <GoogleProtobufPackageVersion>3.5.1</GoogleProtobufPackageVersion>
+    <GoogleProtobufPackageVersion>3.10.1</GoogleProtobufPackageVersion>
     <LightGBMPackageVersion>2.2.3</LightGBMPackageVersion>
     <MicrosoftExtensionsPackageVersion>2.1.0</MicrosoftExtensionsPackageVersion>
-    <MicrosoftMLOnnxRuntimePackageVersion>0.5.1</MicrosoftMLOnnxRuntimePackageVersion>
+    <MicrosoftMLOnnxRuntimePackageVersion>1.0</MicrosoftMLOnnxRuntimePackageVersion>
```

Comment:
This should be `1.0.0`

Reply:
Good catch. Fixed. Thanks!

---
In reply to: [340776622](https://github.com/dotnet/machinelearning/pull/4416#discussion_r340776622) [](ancestors = 340776622)